src/components/ConnectedFilter/index.jsx

```diff
@@ -25,13 +25,15 @@ class ConnectedFilter extends React.Component {
       ? _.union(filterConfigsFields, props.accessibleFieldCheckList)
       : filterConfigsFields;
 
+    this.initialTabsOptions = {};
     this.state = {
       allFields,
       initialAggsData: {},
       receivedAggsData: {},
       accessibility: ENUM_ACCESSIBILITY.ALL,
       adminAppliedPreFilters: Object.assign({}, this.props.adminAppliedPreFilters),
       filter: Object.assign({}, this.props.adminAppliedPreFilters),
```

Comment:
it seems this `this.state.filter` is not useful anymore, could we just remove this, and so that not having both `filter` and `filtersApplied`, you might also want to remove its only reference in `componentDidMount` function